### PR TITLE
New UX Design

### PR DESCRIPTION
**Pull request by Lessica**
- Adjusts layout constraints - I wrote SwiftUI, but it is a piece of shit in my mind.
- Animated background scene - A modified version of ColorfulX with legacy support.
- Localization supports - BartyCrouch

### CODE REVIEW

**Comment by dleovl** [comment]
@haxi0

**Comment by dleovl** [comment]
A UI is already being made (presumably by @haxi0) to be the one chosen to be the UI, please wait for @RootHide

**Comment by dleovl** [comment]
Your UI seemingly removes important checks for compatibility

**Comment by Lessica** [comment]
> A UI is already being made (presumably by @haxi0) to be the one chosen to be the UI, please wait for @RootHide

An extra option. Use whatever you like.

**Comment by Lessica** [comment]
> Your UI seemingly removes important checks for compatibility

Now fixed.

**Comment by dleovl** [comment]
> > A UI is already being made (presumably by @haxi0) to be the one chosen to be the UI, please wait for @RootHide
> 
> An extra option. Use whatever you like.

If yours doesn't get pushed to main, would love to have alternatives.

File: .github/workflows/Bulid.yml
Comment (dleovl) [inline .github/workflows/Bulid.yml:1]: Thanks for using my yml!

**Comment by roothide** [comment]
you may fork the project~

**Pull request closed without merging**